packages/grafana-data/src/utils/namedColorsPalette.ts

```diff
@@ -115,6 +133,33 @@ export function getColorForTheme(color: string, theme: GrafanaTheme): string {
   return (colorsMap[color as Color] = tinycolor(color).toHexString());
 }
 
+export function getColorForTheme2(color: string, theme: GrafanaThemeV2): string {
```

Comment:
instead of creating a separate function, I think you can just make getColorForTheme(color: string, theme: GrafanaTheme | GrafanaThemeV2) , so it accepts both

Reply:
Alright, will do. Thanks!